README.md

```
@@ -0,0 +1,1506 @@
+-   **Testing**: If applicable, add or update tests to cover your changes.
+
+## 📝 License
+
+This project is licensed under the MIT License.
+
+## ✍️ Author Info
+
+-   **Amnesia2k**
+    -   LinkedIn: [https://linkedin.com/in/amnesia2k](https://linkedin.com/in/amnesia2k)
+    -   Twitter: [@amnesia2k](https://twitter.com/amnesia2k)
+    -   Portfolio: [https://your-portfolio.com](https://your-portfolio.com) (Placeholder)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)